src/Controllers/Common/GroupController.cs

```diff
@@ -1,11 +1,63 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 using sodoff.Attributes;
 using sodoff.Model;
 using sodoff.Schema;
 using sodoff.Util;
+using GroupMember = sodoff.Model.GroupMember;
 
 namespace sodoff.Controllers.Common;
+
 public class GroupController : Controller {
+    private static readonly List<RolePermission> RolePermissions;
+
+    static GroupController() {
+        RolePermissions = new List<RolePermission>();
```

Comment:
I wouldn't put this inside a static constructor, but statically initialize the List. I think it also needs explanation because it's not clear what it's for and why it's in this form without knowing how the clan system works